test/config/integration/server_xds.bootstrap.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
1. Yep, there's ASCII protobuf, and we can support this. I opted for JSON as a starting point, as we will definitely need to retain backwards compatibility, I've filed https://github.com/lyft/envoy/issues/1384 to track.

2. Not as it stands, see above comment. It makes things a bit more complicated, more code paths and/or untangling subscription objects. We could probably shrink it down to 2 files eventually: bootstrap and an ADS proto file.